extension/js/background_page/migrations.ts

```diff
@@ -38,13 +40,30 @@ const addKeyInfoFingerprints = async () => {
 };
 
 export const migrateGlobal = async () => {
-  const globalStore = await GlobalStore.get(['key_info_store_fingerprints_added']);
+  const globalStore = await GlobalStore.get(['key_info_store_fingerprints_added', 'local_drafts']);
   if (!globalStore.key_info_store_fingerprints_added) {
     console.info('migrating KeyStorage to add fingerprints and emails of each key...');
     await addKeyInfoFingerprints();
     await GlobalStore.set({ key_info_store_fingerprints_added: true });
     console.info('done migrating');
   }
+  // migrate local drafts (#3337)
+  const storageLocal = await storageLocalGetAll();
+  if (typeof globalStore.local_drafts === 'undefined') {
```

Comment:
Perhaps, if `globalStore.local_drafts` is not undefined, we should not call `storageLocalGetAll()`
(as with other migrations, we can consider this migration done and never return to it).
Or, perhaps, we could just load a flag like `local_drafts_moved` to not cause excessive load here?
Will there rarely be more than one draft?

Reply:
> Perhaps, if globalStore.local_drafts is not undefined, we should not call storageLocalGetAll()
(as with other migrations, we can consider this migration done and never return to it).

I went with this suggestion, thank you 👍